### DISCUSSION TITLE
My Christian Playlist as is - I have not compared it to the one you have yet.

### DISCUSSION THREAD

**Discussion by JesusGod-Pope666-Info** [junguler/m3u-radio-music-playlists#27] [General]
I know it is not totally vanilla, but here is what I have collected over weeks and sorted out and hack the planet to get working. Most of them works. Now that I am here, I might as well post it as I might forget the place later. You can incorporate them into your own playlist or whatever.... If anyone wanna do something with it or something. I know it is far from done in regards of information, but I have not been able to listen to +30 stations or somewhat to get an idea of them all and sort everything out. But by all means, use for whatever:

#EXTM3U
#EXTINF:s3.radio.co/s43b65d791/listen - *One for Israel RADIO AGAPE.FM (M.S - Heb.Eng.Oth)
http://s3.radio.co/s43b65d791/listen 
#oneforisrael.org/about-us/radio-agape-fm – OneForIsrael.org/radio – firmisrael.org/local-partners/one-for-israel

#EXTINF:ais-edge102-live365-dal02.cdnstream.com/a42872?listenerId=Live365-AdBlock&aw_0_1st.playerId=Live365&aw_0_1st.skey=1695540867189 - *MenorahMusic.com Internet Radio (Sabbath Music)
http://ais-edge102-live365-dal02.cdnstream.com/a42872?listenerId=Live365-AdBlock&aw_0_1st.playerId=Live365&aw_0_1st.skey=1695540867189
#https://live365.com/station/Menorah-Music-Internet-Radio-a42872 lavada@MenorahMusic.com

#EXTINF:live365.com/station/Messianic-Joy-a81295 – streaming.live365.com/a81295 - *Messianic Joy (OliveTreeCommunitySpokane.com)
http://ais-edge106-live365-dal02.cdnstream.com/a81295 
#Featuring the teaching ministry of Beit Tikvah with Hylan Slobodkin and soon: Torah4Today with Brent Emery.

#EXTINF:stream1.rcast.net/63566 - *HaDerekRadio.com (Heb/Eng)
https://stream1.rcast.net/63566 
#music, teachings, testimonies and programmes, to help bridge the gap between Jew and Gentile.

#EXTINF:ais-edge101-live365-dal02.cdnstream.com/a71993 - Shuv Radio (Heb.Eng.Oth)
http://ais-edge101-live365-dal02.cdnstream.com/a71993
#Including personal testimonies of the Jewish Messiah, teaching pods on various subjects, 
#voices from the past and current global affairs impacting believers worldwide.

#EXTINF:dc3.serverse.com:1120/JesusGod-Pope666.Info - *Jayshua Messianic Radio (Law Obedience Hebrew Roots Judaizing 24/7)
http://dc3.serverse.com:1120/JesusGod-Pope666.Info
#Email@HolyLanguage.com – Izzi Avraham

#EXTINF:sc13.shoutcaststreaming.us:8342/JesusGod-Pope666.Info - Speech: FirstAmendmentRadio.net
http://sc13.shoutcaststreaming.us:8342/JesusGod-Pope666.Info

#EXTINF:162.244.80.106:11019/JesusGod-Pope666.Info - 20 The Countdown Magazine
http://162.244.80.106:11019/JesusGod-Pope666.Info

#EXTINF:usa19.fastcast4u.com:5530/JesusGod-Pope666.Info - Christians Life Radio
http://usa19.fastcast4u.com:5530/JesusGod-Pope666.Info

#EXTINF: 192.241.185.108:8004/JesusGod-Pope666.Info - CleanAir.fm Christian Radio
http://192.241.185.108:8004/JesusGod-Pope666.Info

#EXTINF:server10.reliastream.com:8043/JesusGod-Pope666.Info - DOC Radio - Christian Hits
http://server10.reliastream.com:8043/JesusGod-Pope666.Info

#EXTINF: uk1.internet-radio.com:8370/JesusGod-Pope666.Info - Easy Listening Christian Gospel
http://uk1.internet-radio.com:8370/JesusGod-Pope666.Info

#EXTINF:ais-edge106-live365-dal02.cdnstream.com/a73249 - Speech: Hebrew Institute Live!
https://ais-edge106-live365-dal02.cdnstream.com/a73249
#Buckle up your seat belts and get ready to journey into the wonderful world of Yeshua HaMashiach - Jesus Christ! 
#Our live call-in number is 1(515)606-5317 || Access Code 520255#.

#EXTINF:s3.radio.co/sd16d576db/listen - IWillGatherYou.com (Messianic Speach)
http://s3.radio.co/sd16d576db/listen

#EXTINF:usa4.fastcast4u.com/proxy/lighthousefm?mp=/1 - LighthouseFM.org VOAR-FM
http://usa4.fastcast4u.com/proxy/lighthousefm?mp=/1

#EXTINF:us2.internet-radio.com:8248/JesusGod-Pope666.Info - Loving Grace Radio (some Hebrew)
http://us2.internet-radio.com:8248/JesusGod-Pope666.Info

#EXTINF:188.122.33.223:8000/JesusGod-Pope666.Info - Lutheran Radio UK
http://188.122.33.223:8000/JesusGod-Pope666.Info

#EXTINF:de1.internet-radio.com:8268/JesusGod-Pope666.Info - Power14745 Radio
http://de1.internet-radio.com:8268/JesusGod-Pope666.Info

#EXTINF:us1.internet-radio.com:8107/JesusGod-Pope666.Info - Power Of the Cross Radio
http://us1.internet-radio.com:8107/JesusGod-Pope666.Info

#EXTINF:69.30.243.166:10652/JesusGod-Pope666.Info - Praise broadcasting Network
http://69.30.243.166:10652/JesusGod-Pope666.Info

#EXTINF:66.70.249.70:5200/JesusGod-Pope666.Info - Remnant Nation Radio
http://66.70.249.70:5200/JesusGod-Pope666.Info

#EXTINF:us2.internet-radio.com:8071/JesusGod-Pope666.Info - Safelife Radio
http://us2.internet-radio.com:8071/JesusGod-Pope666.Info

#EXTINF:us1.internet-radio.com:8045/JesusGod-Pope666.Info - S/Chill and Hip Hop
http://us1.internet-radio.com:8045/JesusGod-Pope666.Info

#EXTINF:62.210.24.124:8335/JesusGod-Pope666.Info - Stream???
http://62.210.24.124:8335/JesusGod-Pope666.Info

#EXTINF:27.131.74.8:8168/JesusGod-Pope666.Info - The Light Worship
http://27.131.74.8:8168/JesusGod-Pope666.Info

#EXTINF:5.63.151.52:7032/JesusGod-Pope666.Info - The Power Station
http://5.63.151.52:7032/JesusGod-Pope666.Info

#EXTINF:us1.internet-radio.com:8208/stream - Total freedom network
http://us1.internet-radio.com:8208/stream

#EXTINF:199.184.153.60:8020/JesusGod-Pope666.Info - Truth Triad TruthNetwork.com
http://199.184.153.60:8020/JesusGod-Pope666.Info

#EXTINF:192.241.185.108:8013/JesusGod-Pope666.Info - WTBI.org (World? Tabernacle Baptist International?)
http://192.241.185.108:8013/JesusGod-Pope666.Info

#EXTINF:mp3streaming.planetwideradio.com:9125/ruach?station - Ruach Radio (RuachCityChurch.org/radio)
http://mp3streaming.planetwideradio.com:9125/ruach?station

#EXTINF:edge-audio-01-cr.sharp-stream.com:80/ucbuk.mp3?station - UCB@UCB.co.UK 1 (United Christian Broadcasters)
http://edge-audio-01-cr.sharp-stream.com:80/ucbuk.mp3?station

#EXTINF:edge-audio-01-cr.sharp-stream.com:80/ucbinspirational.mp3?station - UCB@UCB.co.UK 2 (United Christian Broadcasters)
http://edge-audio-01-cr.sharp-stream.com:80/ucbinspirational.mp3?station

#EXTINF:radio.wanderingsheep.net:8021/christianjazz - Christian Smooth Jazz
http://radio.wanderingsheep.net:8021/christianjazz

#EXTINF:streaming.radio.co/sa06221901/listen - Non Christian? - Ivri Shesh Hebrew
http://streaming.radio.co/sa06221901/listen

#EXTINF:176.31.111.65:4918/JesusGod-Pope666.Info - Non Christian - J-AIR.com.au 88FM - Jewish Australia Internet Radio
http://176.31.111.65:4918/JesusGod-Pope666.Info

#EXTINF IsraelNewsTalkRadio.com - IsraelNewsTalkRadio not working
http://dfw.audio-stream.com/proxy/israelne?mp=/live_live

#EXTINF IsraelNewsTalkRadio.com - IsraelNewsTalkRadio not working
https://dfw.audio-stream.com/proxy/israelne?mp=/live_live

#EXTINF:pcr.streamguys1.com/pcrnational-96k.aac - Premier Christian Radio 1332 live - www.premierchristianradio.com - 020 7316 1300
http://pcr.streamguys1.com/pcrnational-96k.aac

#EXTINF:ais-edge103-live365-dal02.cdnstream.com/a76558?listenerId=Live365-AdBlock&aw_0_1st.playerId=Live365&aw_0_1st.skey=1698767206528 - Prophetic Encounter Radio
http://ais-edge103-live365-dal02.cdnstream.com/a76558?listenerId=Live365-AdBlock&aw_0_1st.playerId=Live365&aw_0_1st.skey=1698767206528

#EXTINF:streaming.live365.com/a76558?listenerId=Live365-AdBlock&aw_0_1st.playerId=Live365&aw_0_1st.skey=1698767206528 - Prophetic Encounter Radio
http://streaming.live365.com/a76558?listenerId=Live365-AdBlock&aw_0_1st.playerId=Live365&aw_0_1st.skey=1698767206528

#EXTINF:streaming.live365.com/a76558?listenerId=Live365-AdBlock&aw_0_1st.playerId=Live365&aw_0_1st.skey=1698767206528 - King David Network
https://streaming.live365.com/a93760?listenerId=Live365-AdBlock&aw_0_1st.playerId=Live365&aw_0_1st.skey=1698767515468

#EXTINF:streaming.live365.com/a76558?listenerId=Live365-AdBlock&aw_0_1st.playerId=Live365&aw_0_1st.skey=1698767206528 - King David Network
https://ais-edge102-live365-dal02.cdnstream.com/a93760?listenerId=Live365-AdBlock&aw_0_1st.playerId=Live365&aw_0_1st.skey=1698767515468

#Hebrew playlist: http://fmstream.org/index.php?c=ISR
 
#multiculturalism is not a value - illigal immigrants are criminals - illigal migrants

**Comment by junguler** [comment]
i appreciate the links but what's with the weird hashtags? this is not a political repo ...

**Reply by JesusGod-Pope666-Info** [reply]
oh the last bit was just some notes I put in yesterday, I just copied and pasted all of the file. I'm sure you can sort things out. :)

A lot of those "Christian" radio channels you had in your Christian file by the way does not seem to work. At least I have a good amount already deleted as they don't work for me in the Audio player... by the way.... you might like it.... you can have a look here: http://am.jesusgod-pope666.info

The files had an issue with only showing the kb as the - was not there as the app takes it. Had to put it in all the places so one can see what kind of radio station it is.

![Screenshot_2023-11-01_01-54-56](https://github.com/junguler/m3u-radio-music-playlists/assets/90806094/7bc8b810-62a8-4360-850e-7d0f00d36cf9)

**Reply by junguler** [reply]
i understand, but i'm not really about pick and choosing what works or not, that's the job of the website owners and admins, all i do is scrape data and push updates to the repo, also keep in mind that not all links work on all countries, some of restricted to only work within a certain area

**Reply by JesusGod-Pope666-Info** [reply]
Yea it is terrible with all those walls on things... Sometimes I have been able to get around it with some of those radio channels :) Hack the Planet. But not always.

**Comment by junguler** [comment]
i will go over the links you've posted and will probably append them to the +merged+ folder when it's time to push an update to it

**Reply by JesusGod-Pope666-Info** [reply]
Yea not sure how you do it - probably have some automated stuff as well to help you out - or are you testing all the radio stations manual for working?

Just a thing, the aM player I am using needs the - before the name begins for putting the name in the playlist. That was a huge problem with near all the radio stations with the file. Don't know much about how everything else works but... That was an issue.

**Reply by JesusGod-Pope666-Info** [reply]
#EXTINF IsraelNewsTalkRadio.com - IsraelNewsTalkRadio not working
http://dfw.audio-stream.com/proxy/israelne?mp=/live_live

does not work, and there might be some others - but overall, pretty near everything works. But work in progress - now testing your radio stations from the Christian file. Have already sorted around 20 or so that does not work for me.

**Reply by junguler** [reply]
no i don't do manual checking, it would be insane because i have over 500k unique links listed here, i have a [script](https://github.com/junguler/m3u-radio-music-playlists/blob/main/stuff/check-for-dead.sh) that does the checking but then again it's just there for people who want it, i'm not using it because like i said it would be too much time and effort

**Reply by JesusGod-Pope666-Info** [reply]
Yea, just what I already have, is taking a lot of time to sort and look through. A lot longer then expected to fix.
Interesting with the scripts I wonder if I can use it... although I really should stop this radio thing and get on moving on other stuff.

Yea not sure how you gather and do all your stuff.

**Reply by junguler** [reply]
indeed it is very time consuming, what i usually do is spent 2-3 days on a new website, find how it's set up and how/if i can scrape it, push the update to this repo and don't think about it for a week or so

as for the script you can easily run it via a github repo and their github actions, i can show you a basic usage example if you are interested but it'll take time for it to complete and it would be restricted to the ip of the virtual machine github uses (most likely american ip since they use their own azure solutions)

or if you have access to a linux shell, on windows it's usually via cygwin, msys2 or wsl(2), you can run the script locally and get results suited for your area

edit: oops i see that you are on linux already, ignore the part about windows
